docs/structure-samples/python/python.md

```diff
@@ -0,0 +1,50 @@
+## Python project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a Python standard application.
+
+```bash
+
+~root
+│
+├── .devcontainer.json              
```

Comment:
Typically we have a .devcontainer folder so that we can have a Dockerfile or docker-compose.yaml optionally. But maybe this is simpler? Never tried without a folder.

Reply:
No, great catch, that was actually me going fast. Fixed for all templates. Thank you.